src/content/developers/docs/consensus-mechanisms/pos/block-proposal/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Block proposal
+description: Explanation of how blocks are proposed in proof-of-stake Ethereum.
+lang: en
+---
+
+Blocks are the fundamental units of the blockchain. This page explains how they are produced.
+
+## Prerequisites {#prerequisites}
+
+Block production is part of the proof-of-stake protocol. To help understand this page, we recommend you read about [proof-of-stake](src/content/developers/docs/consensus-mechanisms/pos/) and [block architecture](src/content/developers/docs/blocks/).
+
+## Who produces blocks? {#who-produces-blocks}
+
+Blocks are produced by validators. Validators are node operators that run validator software as well as their execution and consensus clients, and have deposited at least 32 ETH into the deposit contract. However, each individual validator is only occasionally responsible for producing a block. Ethereum measures time in slots and epochs. Each slot is twelve seconds, and 32 slots (6.4 minutes) make up an epoch. Every slot is an opportunity to add a new block on Ethereum.
```

Comment:
Would be careful between using "produced" vs "proposed"... validators are the ones who propose blocks, but blocks could be "produced" by anyone

Would also argue, "validators" (an account, set of keys) are not "node operators" (people who run the hardware/sotware).

```suggestion
Blocks are proposed by validator accounts. Validator accounts are managed by node operators who run validator software as part of their execution and consensus clients, and have deposited at least 32 ETH into the deposit contract. However, each individual validator is only occasionally responsible for proposing a block. Ethereum measures time in slots and epochs. Each slot is twelve seconds, and 32 slots (6.4 minutes) make up an epoch. Every slot is an opportunity to add a new block on Ethereum.
```